linux/command_tools.bsh

```diff
@@ -92,6 +92,7 @@ function get_array_from_environment()
 # .. function:: parse-generic
 #
 # :Arguments: ``$1``.. - Arguments to be sent to some command
+# :Parameters: [``arguments_with_value``] - An array of command arguments that must take one argument
```

Comment:
I forget, are we capitalizing these parameters?

Reply:
It does depends, but in this case I think no makes sense. For started, it's not exported